LunarTimes/AddLocationTableViewController.swift

```diff
@@ -0,0 +1,209 @@
+//
+//  AddLocationTableViewController.swift
+//  Sunrise & Sunset
+//
+//  Created by Angel Colon-Ramirez on 3/28/20.
+//  Copyright © 2020 LetsHangLLC. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+import Alamofire
+import SwiftyJSON
+import GoogleMobileAds
+import LocationPicker
+import LhHelpers
+import DatePickerDialog
+
+class AddLocationViewModel {
+    
+}
+
+
+class AddLocationTableViewController: UITableViewController, BaseViewController {
+    var viewModel: AddLocationViewModel!
+    
+    static var storyboardName = "Main"
+    
+    static var viewControllerIdentifier = "AddLocationTableViewController"
+    
+    typealias BaseViewModel = AddLocationViewModel
+    
+    
+    var placemark: CLPlacemark?
+    var latitude = 70.0;
+    var longitude = 70.0;
+    var address = ""
+    var delegate: LocationSelectedDelegate?
+    let defaults = UserDefaults.standard
+    var displayLocationDelegate: DisplayClickedLocationDelegate?
+    
+    var locationDict = [String:[Double]] ()
```

Comment:
Add `SunriseLocation` or other class to use the data from LocationPicker's to make it NSCodable or whatever it needs for defaults